### DISCUSSION TITLE
GitHub Education benefits still locked

### DISCUSSION THREAD

**Discussion by Si0077-cloud** [community/community#146899] [GitHub Education]
### Hi everyone,

I trust everyone is doing great , my request for github student pack is approved on 9th Dec but till this day  the benefits that come with are locked , what should do ?

**Comment by Mohrip** [comment]
> ### Hi everyone,
> I trust everyone is doing great , my request for github student pack is approved on 9th Dec but till this day the benefits that come with are locked , what should do ?

Hi,

If your GitHub Education benefits are still locked, try the following steps:

Check Email: Ensure you've received the confirmation email for the approval of your GitHub Student Pack and follow any further instructions in the email.

Refresh Verification: Go to your GitHub Education settings and refresh the verification process by re-uploading your student ID or other required documents.

Wait a Few Days: Sometimes there can be a delay in processing the request. If it's only been a few days, the benefits might still be updating.

Contact Support: If the issue persists, reach out to GitHub Support or the GitHub Education team directly via their [contact form](https://support.github.com/contact). Be sure to provide your approval date and any relevant information.

**Comment by davevad93** [comment]
Hi @Si0077-cloud ,

You don't have to do anything else, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Si0077-cloud was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
